<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [HybridSuite]" guice-stage="DEVELOPMENT" verbose="5">
  <listeners>
    <listener class-name="com.framework.testng.api.base.RetryEngine"/>
  </listeners>
  <test thread-count="5" parallel="classes" name="HybridTest(failed)" verbose="5">
    <classes>
      <class name="com.leaftaps.testcases.TC001_Login_Cucumber">
        <methods>
          <include name="startTestCase"/>
          <include name="setUpClass"/>
          <include name="postCondition"/>
          <include name="runScenario" invocation-numbers="0 0"/>
          <include name="setValues"/>
          <include name="tearDownClass"/>
          <include name="startReport"/>
          <include name="preCondition"/>
          <include name="endResult"/>
        </methods>
      </class> <!-- com.leaftaps.testcases.TC001_Login_Cucumber -->
      <class name="com.leaftaps.testcases.TC001_VerifyLogin">
        <methods>
          <include name="runLogin" invocation-numbers="0 0"/>
          <include name="startTestCase"/>
          <include name="setUpClass"/>
          <include name="postCondition"/>
          <include name="tearDownClass"/>
          <include name="startReport"/>
          <include name="setValues"/>
          <include name="preCondition"/>
          <include name="endResult"/>
        </methods>
      </class> <!-- com.leaftaps.testcases.TC001_VerifyLogin -->
    </classes>
  </test> <!-- HybridTest(failed) -->
</suite> <!-- Failed suite [HybridSuite] -->
